apache-activemq-mixin/dashboards/apache-activemq-topic-overview.libsonnet

```diff
@@ -775,11 +775,15 @@ local topicSummaryPanel(matcher) = {
           'Time 4': true,
           '__name__ 1': true,
           '__name__ 2': true,
-          'activemq_cluster 1': true,
```

Comment:
what is this change for?

Reply:
<img width="3349" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/8333d3c1-c166-4a65-93e0-8631d8a4839f">
On the summary table, it will show the cluster and instance. I thought that this would be nice to have on the table, unless you disagree.